CHANGELOG.md

```diff
@@ -208,7 +208,7 @@ $proxyClass = $proxyGenerator->generateProxy(
     new ClassGenerator('ProxyClassName')
 );
 
-eval('<?php ' . $proxyClass->generate());
+eval('<? ' . $proxyClass->generate());
```

Comment:
This should really not use short tags

Reply:
Yep. https://github.com/codelicia/xulieta/issues/19 I can open the `php` tag at the beginning of the code sample as a workaround, or just ignore this one as we don't have the tool here.

Reply:
Which one do you prefer?

Reply:
I think the tool has a bug here: this is a string inside an `eval()` block, but xulieta caught it as a code delimiter

Reply:
Fixed by opening the php tag at the beginning of the code sample. I do have an issue open for that.